source/includes/common-mc-admin-config.rst

```diff
@@ -0,0 +1,978 @@
+
+Specify the maximum limit for undelivered messages. Defaults to ``10000``.
+
+.. end-minio-notify-mysql-queue-limit
+
+.. start-minio-notify-mysql-comment
+
+Specify a comment to associate with the MySQL configuration.
+
+.. end-minio-notify-mysql-comment
+
+
+.. Descriptions for Kafka bucket notification configurations.
+   Used in the following files:
+   - /source/reference/minio-server/minio-server.rst
+   - /source/reference/minio-cli/minio-mc-admin/mc-admin-config.rst
+   - /source/monitoring/bucket-notifications/publish-events-to-kafka.rst
+
+.. start-minio-notify-kafka-enable
+
+Specify ``on`` to enable publishing bucket notifications to a Kafka
+service endpoint.
+
+Defaults to ``off``.
+
+.. end-minio-notify-kafka-enable
+
+.. start-minio-notify-kafka-brokers
+
+Specify a comma-separated list of Kafka broker addresses. For example:
+
+``"kafka1.example.com:2021,kafka2.example.com:2021"``
+
+.. end-minio-notify-kafka-brokers
+
+.. start-minio-notify-kafka-topic
+
+Specify the name of the Kafka topic to which MinIO publishes 
+bucket events.
+
+.. end-minio-notify-kafka-topic
+
+.. start-minio-notify-kafka-sasl-username
+
+Specify the username for performing SASL/PLAIN or SASL/SCRAM authentication
+to the Kafka broker(s).
+
+.. end-minio-notify-kafka-sasl-username
+
+.. start-minio-notify-kafka-sasl-password
+
+Specify the password for performing SASL/PLAIN or SASL/SCRAM authentication
+to the Kafka broker(s).
+
+.. end-minio-notify-kafka-sasl-password
+
+.. start-minio-notify-kafka-sasl-mechanism
+
+Specify the SASL mechanism to use for authenticating to the Kafka broker(s).
+MinIO supports the following mechanisms:
+
+- ``PLAIN`` (Default)
+- ``SCRAM``
+
+.. end-minio-notify-kafka-sasl-mechanism
+
+.. start-minio-notify-kafka-tls-client-auth
+
+Specify the client authentication policy of the Kafka broker(s).
+
+.. https://pkg.go.dev/crypto/tls#ClientAuthType ?
+
+.. end-minio-notify-kafka-tls-client-auth
+
+.. start-minio-notify-kafka-sasl
+
+Specify ``on`` to enable SASL authentication.
+
+.. end-minio-notify-kafka-sasl
+
+.. start-minio-notify-kafka-tls
+
+Specify ``on`` to enable TLS connectivity to the Kafka broker(s)
+
+.. end-minio-notify-kafka-tls
+
+.. start-minio-notify-kafka-tls-skip-verify
+
+Enables or disables TLS verification of the NATS service endpoint TLS
+certificates.
+
+- Specify ``on`` to disable TLS verification (Default).
+- Specify ``off`` to enable TLS verification.
+
+.. end-minio-notify-kafka-tls-skip-verify
+
+.. start-minio-notify-kafka-client-tls-cert
+
+Specify the path to the client certificate to use for performing
+mTLS authentication to the Kafka broker(s).
+
+.. end-minio-notify-kafka-client-tls-cert
+
+.. start-minio-notify-kafka-client-tls-key
+
+Specify the path to the client private key to use for performing
+mTLS authentication to the Kafka broker(s).
+
+.. end-minio-notify-kafka-client-tls-key
+
+.. start-minio-notify-kafka-version
+
+Specify the version of the Kafka cluster.
```

Comment:
@vadmeste @poornas This field puzzled me - maybe it makes sense to Kafka users, but why do we need to specify the version of the Kafka cluster as part of the request?

Reply:
ah - @Praveenrajmani @kannappanr looks like quite a bit of the Kafka target was authored by the both of you. Any insight here?

Reply:
https://github.com/shopify/sarama/blob/v1.20.1/config.go#L327

Reply:
> The version of Kafka that Sarama will assume 

[sarama](https://github.com/Shopify/sarama) is the client pkg here that we use